EpsilonAuxBMS/Src/SetContactorsTask.c

```diff
@@ -179,8 +182,8 @@ void setContactorsTask(void const* arg)
                     state = DISCHARGE_ENABLE_CHECK;
                 }
 
-                common = !HAL_GPIO_ReadPin(COMMON_SENSE_GPIO_Port, COMMON_SENSE_Pin);
-                discharge = !HAL_GPIO_ReadPin(DISCHARGE_SENSE_GPIO_Port, DISCHARGE_SENSE_Pin);
+                uint8_t common = !HAL_GPIO_ReadPin(COMMON_SENSE_GPIO_Port, COMMON_SENSE_Pin);
```

Comment:
What are your thoughts on changing these names to like `commonSense`, `chargeSense`, etc? I'm thinking it would be more descriptive and we wouldn't get confused with the enable LEDs

Reply:
Yeah, sure ting